packages/compile-common/src/profiler/updated.ts

```diff
@@ -1,5 +1,5 @@
 import * as path from "path";
-import * as fse from "fs-extra";
+import * as fs from "fs";
```

Comment:
It's used for the new-to-legacy shims; Codec still runs on the legacy bytecode format.  I don't think there's any way to avoid invoking it.  As for whether this will prevent browser compatibility, uh... you tell me?  I thought there are ways to deal with that?  In the worst case, maybe we can handle that afterward?

Reply:
Sure